blobs.go

```diff
@@ -185,6 +190,16 @@ type BlobWriter interface {
 	Reader() (io.ReadCloser, error)
 }
 
+// BlobKeeper combines the operations needed for registry's upkeeping such as
+// blob access, read, enumeration and delete. Can be used to prune unnecessary
+// objects.
+type BlobKeeper interface {
```

Comment:
@RichardScothern Ok, I'll remove it. Now to provide equivalent interface, two possibilities come to my mind:

1. Extend `Namespace` interface for `Blobs(ctx context.Context) BlobStore` and add missing methods to `storage.blobStore` to complete `BlobStore` interface. The methods would `return ErrUnsupported`.
2. Replace `storage.RegistryBlobKeeper()` with `storage.RegistryBlobEnumerator()`, `storage.RegistryBlobProvider()`, `storage.RegistryBlobDeleter()` and `storage.RegistryBlobStatter()` reurning corresponding interfaces to registry's blob store.

I'd prefer the the 1st, but OTOH giving an impression of implementing interface and returning *unsupported* with considerable number of methods doesn't seem right to me. WDYT?